test/e2e/login.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { test, expect } from '@playwright/test';
+
+test('Login with User 1', async ({ page }) => {
+  await page.goto('/login?redirectTo=%2F');
+  await page.getByRole('button', { name: 'Sign In' }).click();
+  await page.getByLabel('Username').fill('user1');
+  await page.getByLabel('Password').fill('user1pass');
+  await page.getByRole('button', { name: 'Login' }).click();
+  await page.waitForURL('http://localhost:8080');
+  const locator = page.getByText('user1', { exact: true });
+  const starchartHeading = page.getByRole('heading', { name: 'Starchart' });
+  await expect(locator).toContainText('user1');
```

Comment:
Is it possible to use some other locator for the username? 
This reads like expect(element with `this text`) to contain(`this text`)

Reply:
`<p class="chakra-text cha-0">user1</p>` do you have any recommendations on how to select this? It is the user1 that shows up in the top right on the header. 

further down the same class is used on another paragraph

also do you know why the css is wonky when using the codegen script? It will sometimes look different, very frequently will come back like this:

![image](https://user-images.githubusercontent.com/77400826/223504194-9c2d5329-656a-4e78-ae74-d60691b55b83.png)

Reply:
Add an `id` or `class` to the React element so you can query it more easily in the test.